src/Utils.ts

```diff
@@ -2100,7 +2100,7 @@ export class Utils {
             return;
         }
         
-        eagle.setSelection(Eagle.RightWindowMode.Inspector, n, location);
+        eagle.setSelection( n, location);
```

Comment:
Not your fault, but can we change 'n' to 'node' in this function. Just to make it clearer.